acestep/ui/gradio/help_content_i18n_test.py

```
@@ -0,0 +1,83 @@
+        "btn_label",
+        "close_label",
+        "getting_started",
+        "service_config",
+        "generation_simple",
+        "generation_custom",
+        "generation_remix",
+        "generation_repaint",
+        "generation_extract",
+        "generation_lego",
+        "generation_complete",
+    }
```

Ruff (0.15.2)

[warning] Mutable default value for class attribute

(RUF012)